client/me/profile/index.jsx

```diff
@@ -121,18 +113,26 @@ class Profile extends Component {
 							/>
 						</FormFieldset>
 
-						<FormFieldset
-							className={ clsx( {
-								'profile__gravatar-fieldset-is-loading': this.props.isFetchingUserSettings,
-							} ) }
-						>
-							<ToggleControl
-								disabled={ this.props.getDisabledState() || this.props.isFetchingUserSettings }
-								checked={ this.props.getSetting( 'gravatar_profile_hidden' ) }
-								onChange={ this.toggleGravatarHidden }
-								label={ <UpdatedGravatarString gravatarProfileLink={ gravatarProfileLink } /> }
-							/>
-						</FormFieldset>
+						<p className="profile__gravatar-profile-description">
+							<span>
+								{ this.props.translate(
+									'Your WordPress.com profile is powered by Gravatar. Your Gravatar is public by default and may appear on any site using Gravatar when you’re logged in with {{strong}}%(email)s{{/strong}}.' +
```

Comment:
```suggestion
									'Your WordPress.com profile is connected to Gravatar. Your Gravatar is public by default and may appear on any site using Gravatar when you’re logged in with {{strong}}%(email)s{{/strong}}.' +
```

Given so few fields actually sync with Gravatar, I think it's better to say "connected to" over "powered by"

Reply:
Thank you for the feedback. I have addressed it through [this commit](https://github.com/Automattic/wp-calypso/pull/93023/commits/a58353e765266b9a34099adc6fae155248f36899).